angular/projects/spark-angular/src/lib/components/sprk-card/sprk-card.stories.ts

```diff
@@ -124,24 +924,25 @@ export const highlightedHeader = () => ({
           signiferumque at. Numquam.
         </p>
       </div>
-  </sprk-card>`,
+    </sprk-card>
```

Comment:
missing `sprkText`

Reply:
@bobwise that is a legacy story, the old code API.